qa/nose_wrapper/__main__.py

```diff
@@ -1,4 +1,86 @@
 import sys
+
+if sys.version_info >= (3, 12):
+    from importlib import machinery, util
```

Comment:
This code is duplicated twice, here and in the dali/test/python/nose_utils.py

Can you place it in just one file as something like `utils/imp_backport.py` and import it in both places?

Reply:
Done